### PR TITLE
Fix local 'gulp' error; implement ability to enter custom pedals/boards in inches and millimeters

### LINKED ISSUE #732: Allow entry of custom dimensions in mm (as well as inches)
Because we're too lazy to do the unit conversions outside the great tool.

### LINKED ISSUE #868: Local Build Issue
I'm on an M1 MacBook Pro, and run the commands outlined in the README in a Rosetta Terminal (so I have side-by-side x86 and ARM Brew installations). 

Here's my output when running `gulp`. I'm running this off of my fork:

`❯ gulp
[21:22:30] Using gulpfile ~/Dev/git/PedalPlayground.github.io.cbserra/gulpfile.js
[21:22:30] Starting 'styles'...
[21:22:30] Starting 'scripts'...
[21:22:30] Starting 'watch'...
[21:22:30] Finished 'watch' after 15 ms
[21:22:30] gulp-notify: [Task Failed [gulp-sass]] See console.
TASK: [gulp-sass]
PROB: bower_components/bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss
Error: Invalid CSS after "...     floor(math": expected expression (e.g. 1px, bold), was ".div($grid-gutter-w"
        on line 369 of bower_components/bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss
>> $navbar-padding-horizontal:        floor(math.div($grid-gutter-width, 2)) !def
   -----------------------------------------^

[21:22:30] Finished 'styles' after 95 ms
[21:22:32] gulp-notify: [Gulp notification] JS Compiled!
[21:22:32] /Users/cbserra/Dev/git/PedalPlayground.github.io.cbserra/public/scripts/scripts-min.js.gz reloaded.
[21:22:32] gulp-notify: [Gulp notification] JS Compiled!
[21:22:32] /Users/cbserra/Dev/git/PedalPlayground.github.io.cbserra/public/scripts/scripts.js.gz reloaded.
[21:22:32] Finished 'scripts' after 1.93 s
[21:22:32] Starting 'default'...
[21:22:32] Finished 'default' after 72 μs
^C
`

First, I get that `gulp-sass` error. Even after running `sudo npm rebuild node-sass -g`, the error is present.

Also, the script never completes. I have to Ctrl+C to exit.

### PR DESCRIPTION
**Pull request by cbserra**
- Update bootstrap-sass in bower to eliminate runtime error. Added .nvmrc file to more easily set node version for workspace. Closes #868 
- Implemented toggle switch to choose between inches and millimeters when inputting custom pedals and pedalboards. Closes #732 

1. New toggle switch under **Width** and **Height** input fields allows the user to switch between inches and millimeters. Inches will always be default.
Screen Shot 2022-04-06 at 3 27 27 AM

2. To add a custom item in millimeters, click the toggle to switch to `mm.` and enter Width and Height values.
Screen Shot 2022-04-06 at 3 28 07 AM

3. An example custom pedalboard created, with the dimension values converted to inches.
Screen Shot 2022-04-06 at 3 28 38 AM

4. The same toggle switch was added to adding a Custom Pedal, as well.
Screen Shot 2022-04-06 at 3 29 26 AM

5. An example custom pedal created, with the dimension values converted to inches.
Screen Shot 2022-04-06 at 3 29 40 AM

6. Screenshot of site with two custom items.
Screen Shot 2022-04-06 at 3 30 06 AM

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -9,12 +9,13 @@
...
 	"ignore": ["**/.*", "node_modules", "bower_components", "test", "tests"],
 	"dependencies": {
-		"bootstrap-sass": "^3.3.7",
+		"bootstrap-sass": "3.4.1",
 		"modernizer": "^2.8.2",
 		"sass-flex-mixin": "^1.0.3",
 		"jquery-zoom": "^1.7.18",
 		"draggabilly": "^2.1.1",
 		"jquery": "3.1.1",
+		"jquery-ui": "1.10.3",
```
Comment (cbserra) [inline bower.json:18]: jquery.switchButton plugin required `jquery-ui` library.

**Review by cbserra — COMMENTED** [review]
Added questions about added jQuery plugin file locations

**Review by cbserra — COMMENTED** [review]
Approved. @tehtrav, @ArislanShiva any feedback?

**Comment by dmitry-mashkov** [comment]
Hi @tehtrav, can this be moved already? The Millimeters switcher is really wanted

**Pull request closed without merging**